documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md

```diff
@@ -0,0 +1,27 @@
+# API Readiness Checklist
+
+Checklist for Carrier Billing Refund v0.1.0 in r1.1
```

Comment:
The actual API Version number could also been set within the release PR, but as you asked for review here:

```suggestion
Checklist for Carrier Billing Refund v0.1.0-rc.1 in r1.1
```